src/SFML/Graphics/Font.cpp

```diff
@@ -388,7 +377,7 @@ float Font::getKerning(std::uint32_t first, std::uint32_t second, unsigned int c
     if (first == 0 || second == 0)
         return 0.f;
 
-    auto* face = m_fontHandles ? m_fontHandles->face.get() : nullptr;
+    FT_Face face = m_fontHandles ? m_fontHandles->face : nullptr;
```

Comment:
`auto` here was obfuscating the original type, it is not obvious from the initialization expression.